AdobeMediaGallery/etc/module.xml

```diff
@@ -6,5 +6,10 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_AdobeMediaGallery" />
+    <module name="Magento_AdobeMediaGallery">
+        <sequence>
+            <module name="Magento_Catalog"/>
```

Comment:
moule.xml is used to define installation dependencies. `AdobeMediaGallery` depends on Catalog and CMS modules, but they do not necessarily need to be installed before `AdobeMediaGallery` (as there are not database schema dependency). So these dependencies should only be added to composer.json

Reply:
@sivaschenko thank you for an explanation! I got the point now with this soft/hard dependencies in our case :)